### PR TITLE
chore: tweak the output of get scheme (support HTTP1.1)

### PR DESCRIPTION
**Pull request by tzssangglass**
Signed-off-by: tzssangglass <tzssangglass@gmail.com>

### CODE REVIEW

File: src/http/ngx_http_wasm_api.c [hunk 1]
```diff
@@ -918,8 +918,14 @@ ngx_http_wasm_req_get_header(ngx_http_request_t *r, char *key,  int32_t key_size
                     break;
 
                 case PROXY_WASM_HEADER_SCHEME:
-                    val = r->schema.data;
-                    val_len = r->schema.len;
+                    if (r->connection->ssl) {
```
Comment (spacewander) [inline src/http/ngx_http_wasm_api.c:921]: We can refactor the code and use getter instead?

Comment (tzssangglass) [reply src/http/ngx_http_wasm_api.c:921]: Do you mean to refactor the logic used above as well?

**Review by membphis — APPROVED** [review]
LGTM

File: src/http/ngx_http_wasm_api.c (hunk 1 above)
Comment (tzssangglass) [reply src/http/ngx_http_wasm_api.c:921]: do it later.

Comment (tzssangglass) [reply src/http/ngx_http_wasm_api.c:921]: will be done soon

File: src/http/ngx_http_wasm_api.c [hunk 2]
```diff
@@ -918,8 +918,14 @@ ngx_http_wasm_req_get_header(ngx_http_request_t *r, char *key,  int32_t key_size
...
+                        val = scheme_https.data;
+                        val_len = scheme_https.len;
+                        break;
```
Comment (spacewander) [inline src/http/ngx_http_wasm_api.c:924]: It is weird to use break in an if block

Comment (tzssangglass) [reply src/http/ngx_http_wasm_api.c]: update

File: src/http/ngx_http_wasm_api.c [hunk 3]
```diff
@@ -921,11 +921,10 @@ ngx_http_wasm_req_get_header(ngx_http_request_t *r, char *key,  int32_t key_size
                     if (r->connection->ssl) {
                         val = scheme_https.data;
                         val_len = scheme_https.len;
-                        break;
+                    }else {
```
Comment (spacewander) [inline src/http/ngx_http_wasm_api.c]:
```suggestion
                    } else {
```

**Review by spacewander — APPROVED** [review]

File: src/http/ngx_http_wasm_api.c (hunk 3 above)
Comment (tzssangglass) [reply src/http/ngx_http_wasm_api.c]: update

**Pull request merged**